docs/patterns/logo-wall/index.md

```diff
@@ -25,7 +25,7 @@ A logo wall is a visual arrangement of logos representing various brands, compan
 <div class="grid">
   <div>
     <h3>Bordered</h3>
-    <uxdot-example>
+    <uxdot-example width-adjustment="1010px">
```

Comment:
For this image, the intrinsic size is listed as `1120x110`. Needs updated here and on the `<img>` tag.

Reply:
`1010px` is the maximum size that `uxdot-example` has room for given desktop layout with padding.   I updated the `width` attribute on the `<img>` tag.

My assumption here is that setting the `width-adjustment` on the uxdot-example will decrease possibility of any CLS, but the `max-width: 100%` might already be handling that on its on given the intrinsic size is larger then the available space.  🤔